Omnibus.md

```diff
@@ -3446,16 +3471,18 @@ Restore Enchantment
   spell is wasted if the quest fails or if permission for a repair is outright
   denied.
 
-### Regenerate the Scalp (5th Circle) ### {#regenerate-the-scalp}
+### Regenerate the [Soul]{.new} (5th Circle) ### {#regenerate-the-soul}
 
 __Uses:__ 1 - __Verbal:__ 30 words - __Material:__ An item similar to the
-spellcaster's scalp, with some sort of indication that it is their second scalp
-- __Caveats:__ [Regeneration](#regeneration-caveat), [Advanced
-Regeneration](#advanced-regeneration)
+spellcaster's [soul token]{.new}, with some sort of indication that it is
+their second [soul token]{.new} -
+__Caveats:__ [Regeneration](#regeneration-caveat),
```

Comment:
Not important, but `__Caveats:__` fits on the previous line.

Reply:
Caveats moved to previous line, regeneration and advanced regeneration are broken up though (having both on the same line is 85 characters).